components/style/properties.mako.rs

```diff
@@ -2665,6 +2734,19 @@ pub mod shorthands {
             _ => Err(()),
         }
     </%self:shorthand>
+
+    <%self:shorthand name="overflow" sub_properties="overflow-x overflow-y">
+        use properties::longhands::{overflow_x, overflow_y};
+
+        let overflow = match input.try(|input| overflow_x::parse(context, input)) {
```

Comment:
`Parser::try` is not needed when you propagate `Err(_)` to the caller (as you do here).